sass/themes/cr/2019/components/navigation/expanded-footer.twig

```diff
@@ -63,11 +63,8 @@
               <a href="/" title="A Second Menu" aria-expanded="false" aria-haspopup="true" data-drupal-link-system-path="<front>" class="is-active"><span class="menu-item__text">Hear From Us</span></a>
               <ul class="menu menu--level-0 menu menu--expanded-footer menu--has-2-items menu--level-1 clearfix">
                 <li class="menu-item menu-item--1 clearfix">
-                  <a class="js-esu-popup" href="#">
+                  <a href="#" data-drupal-link-system-path="<front>" class="is-active">
```

Comment:
We can get rid of the Drupal stuff from here, and probably the active class too

Reply:
Sure